src/modelfree/train.py

```diff
@@ -173,6 +181,22 @@ def sac(batch_size, learning_rate, **kwargs):
                    batch_size=batch_size, learning_rate=learning_rate, **kwargs)
 
 
+@train_ex.capture
+def gail(batch_size, expert_dataset_path, **kwargs):
+    import matplotlib
+    matplotlib.use('pdf')  # ExpertDataset needs this and we don't have tkinter
```

Comment:
Can you also in the PR for GAIL get rid of the `matplotlib` dependency for `ExpertDataset`? It's just used in the `plot` function, which TBH would probably be fine to delete, but if we want to keep it then easy to make that function import matplotlib only when needed. Then we can remove this hack.